app/src/ui/multi-commit-operation/choose-branch/merge-choose-branch-dialog.tsx

```diff
@@ -0,0 +1,212 @@
+import React from 'react'
+import {
+  getAheadBehind,
+  mergeTree,
+  revSymmetricDifference,
+} from '../../../lib/git'
+import { promiseWithMinimumTimeout } from '../../../lib/promise'
+import { Branch } from '../../../models/branch'
+import { ComputedAction } from '../../../models/computed-action'
+import { MergeTreeResult } from '../../../models/merge'
+import { MultiCommitOperationKind } from '../../../models/multi-commit-operation'
+import { ActionStatusIcon } from '../../lib/action-status-icon'
+import { BaseChooseBranchDialog } from './base-choose-branch-dialog'
+
+export abstract class MergeChooseBranchDialog extends BaseChooseBranchDialog {
```

Comment:
This one shouldn't be abstract, right?
```suggestion
export class MergeChooseBranchDialog extends BaseChooseBranchDialog {
```

Interestingly, TypeScript doesn't complain if you instance this class using JSX :til:

Reply:
You are correct! Copy/Pasta mistake. Thanks!